src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3187,6 +3210,25 @@ export class ResultsViewPageStore {
         []
     );
 
+    @computed get ensemblLink() {
+        return currentReferenceGenemeBuild(this.studies.result) ===
+            AppConfig.serverConfig.genomenexus_url_grch38
+            ? AppConfig.serverConfig.ensembl_transcript_grch38_url
+            : AppConfig.serverConfig.ensembl_transcript_url;
+    }
+
+    @computed get genomeNexusClient() {
+        return new GenomeNexusAPI(
+            currentReferenceGenemeBuild(this.studies.result)
```

Comment:
this should be in a computed property @computed referenceGenomeBuild() that references this.studies.

Reply:
and then this will simple reference that computed value instead of calculating it here (and again below).  It's not the calculation is slow, it's that conceptually we want the source of truth to live in a place that is clear to everyone.

Reply:
also, note the typo "GenemeBuild"